content/api/game-state.md

```diff
@@ -36,3 +36,17 @@ r.zone
 
 During a competition match, a USB drive will be used to tell your robot
 which corner it's in. By default during development, this is `0`.
+
+## Start Signal
```

Comment:
In the simulator, this doesn't really exist.

Reply:
ok, in which case its gone!